flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontend.java

```diff
@@ -292,20 +267,14 @@ protected int run(String[] args) {
 	 *
 	 * @param args Command line arguments for the info action.
 	 */
-	protected int info(String[] args) {
+	protected int info(String[] args) throws CliArgsException, FileNotFoundException, ProgramInvocationException {
 		LOG.info("Running 'info' command.");
 
-		// Parse command line options
-		InfoOptions options;
-		try {
-			options = CliFrontendParser.parseInfoCommand(args);
```

Comment:
`CliFrontendParser.parseInfoCommand()` is no longer in use. Can be deleted.

Reply:
Will remove it.